services/QuillLMS/app/controllers/stripe_integration/subscription_checkout_sessions_controller.rb

```diff
@@ -55,7 +65,15 @@ def create
     end
 
     private def success_url
-      "#{subscriptions_url}?checkout_session_id={CHECKOUT_SESSION_ID}"
+      return "#{subscriptions_url}?checkout_session_id={CHECKOUT_SESSION_ID}" if schools.empty?
+
+      "#{teacher_admin_subscriptions_url(school_id: school_ids.first)}&checkout_session_id={CHECKOUT_SESSION_ID}"
+    end
```

Comment:
If a teacher associated with a school, but who is not an admin for that school, renews that school's subscription from their My  Subscriptions page, I think this code is going to send them to the school subscriptions page, which will error because it's only available to admins. Let me know if I'm reading this incorrectly, but otherwise I think we'll have to handle this differently - maybe we can just check to see if they are an admin?

Reply:
Good catch.  I will fix that to check for admin.